content/content-strategy/define-user-content-needs/attribution.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: intros/intro_with_nav
+subtitle: Get to know your users and create artefacts that help you action what you’ve learned.
+category: Content strategy
+---
```

Comment:
Just noticed the file is called 'attribution'. Are there plans to add the author attribution and review date at a later point?

Reply:
no (trisha)

Reply:
Steve. I don't think so. This was DTA only.

Reply:
The attribution.md file md file should be called intro.md